### DISCUSSION TITLE
2010/12/19/%E8%B1%86%E8%B1%86%E7%9A%84%E5%90%8D%E5%AD%97

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#307] [Announcements]
# 2010/12/19/%E8%B1%86%E8%B1%86%E7%9A%84%E5%90%8D%E5%AD%97

起名也是需要灵感的，有时灵光一现，就想到了一个满意的名字。

https://blog.qizhen.xyz/2010/12/19/%E8%B1%86%E8%B1%86%E7%9A%84%E5%90%8D%E5%AD%97

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Bright Wang
2011年01月28日 16:31 编辑
我的儿子也叫豆豆,跟你家豆豆差不多大,10月底出生的.
叫豆豆是因为我老婆爱吃豆皮,豆干之类的,呵呵.

回复
